openslides/motions/static/templates/motions/motion-list.html

```diff
@@ -21,78 +21,173 @@ <h1 translate>Motions</h1>
 
 <div class="row form-group">
   <div class="col-sm-8">
-    <!-- TODO: add filters -->
+    <form class="form-inline">
+      <!-- delete mode -->
+      <div os-perms-lite="motions.can_manage" class="form-group">
+        <label for="deleteSwitcher" translate>Delete mode</label>
+        <switch id="deleteSwitcher" ng-model="isDeleteMode" ng-change="uncheckAll()"
+            on="{{'On'|translate}}" off="{{'Off'|translate}}"
+            class="green wide form-control">
+        </switch>
+      </div>
+      <!-- delete button -->
+      <a ng-show="isDeleteMode && (motions|filter:{selected:true}).length > 0"
+          os-perms="motions.can_manage" ng-click="delete()"
+          class="btn btn-primary btn-sm form-control">
+        <i class="fa fa-trash fa-lg"></i>
+        <translate>Delete selected motions</translate>
+      </a>
+      <!-- TODO: add filters -->
+      &nbsp;
+      <select ng-model="stateFilter" class="form-control" id="stateFilter">
+        <option value="" translate>--- Select state ---</option>
+        <!-- TODO: use default workflow id from config('motions_workflow') -->
+        <option ng-repeat="state in workflows[0].states" value="{{ state.id }}">{{ state.name }}</option>
```

Comment:
> During an event motions might be in different workflows. So your filter has
> to loop over all possible states.

Then the filter is not very useable because to many (unordered) state entries. I 
prefer to use only the selected (default) workflow. I can NOT imagine that a 
manager change the workflow during event. (I know it is possible in OS!)